src/sdk/model/common/snapshot/snapshot-create-request.ts

```diff
@@ -0,0 +1,77 @@
+/* istanbul ignore next: autogenerated */
```

Comment:
should not be above imports, should be just above class declaration

Reply:
fixed